src/hydra/dataProvider.ts

```diff
@@ -679,7 +681,11 @@ function dataProvider(
       ((!!result.total && result.data.length < result.total) ||
         result.pageInfo?.hasNextPage)
     ) {
-      pageParams.pagination.page += 1;
+      if (pageParams.pagination) {
+        pageParams.pagination.page += 1;
+      } else {
+        pageParams.pagination = { page: 2, perPage: 25 };
```

Comment:
```suggestion
        pageParams.pagination = { page: 1, perPage: 25 };
```
?

Reply:
No, I think 2 is the right value in that case.